src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -26,7 +26,7 @@ class Cinema4DNotRunningError(Exception):
     pass
 
 
-_FIRST_CINEMA4D_ACTIONS = ["scene_file", "take"]
+_FIRST_CINEMA4D_ACTIONS = ["scene_file", "take", "output_path", "multi_pass_path"]
```

Comment:
Nit: Should it be `multi_pass_output_path` instead so that it signifies that its an output path?

Reply:
My experience is the term "multi-pass" is unambiguously an output path in Cinema4D. The c4d api uses the handle `RDATA_MULTIPASS_FILENAME`. We can certainly change this line but it would also mean changing the respective action method, the adaptor schema, the init data and the template --  and probably the render settings data class and the ui widget for consistency sake too.

Reply:
Ok thanks for the clarifications!